README.adoc

```diff
@@ -85,6 +85,24 @@ That Makefile target takes care of several steps that can be executed separately
 * create `CatalogSource` and `ConfigMap` with `ClusterServiceVersion` and all CRDs in the `openshift-marketplace`: `$ make deploy-csv`
 * and as the last step the actual installation via creating `OperatorGroup` and `Subscription` in the test namespace.
 
+== Releasing operator
+
+The releases of the operator are automatically managed via Tekton pipeline running under `codeready-toolchain` user account in CRT cluster.
+
+The definition of the CD pipeline can be found here https://github.com/codeready-toolchain/toolchain-cicd.
+The pipeline triggers Makefile targets located in this repository.
+
+=== Broken release
+
+If there is any broken release that cannot be built & pushed through the pipeline and has to be done/fixed manually then do the following steps:
```

Comment:
Same question as in the member-operator PR : https://github.com/codeready-toolchain/member-operator/pull/230#discussion_r543424877

Reply:
done in ea23958